addons/delivery/data/delivery_data.xml

```diff
@@ -16,9 +16,8 @@
             <field name="invoice_policy">order</field>
         </record>
         <record id="free_delivery_carrier" model="delivery.carrier">
-            <field name="name">Free delivery charges</field>
+            <field name="name">Standard delivery</field>
             <field name="fixed_price">0.0</field>
-            <field name="free_over" eval="True"/>
             <field name="amount">1000</field>
```

Comment:
No, we don't. Let's change that for new users. It's only to prevent misuses.
The reason for this change is that, the `fixed_price` being 0, there is no reason to set a "free over X". And we rely on the default value for this boolean, `False`. This means that the view will show an unchecked box, and hide the `amount` above which it could be free (if the box is checked afterwards). But then, if the user changes the `fixed_price`, then fills the box, he would probably like to avoid the default value, zero, because this would give any delivery for free.

Reply:
It's a PO decision to keep the modified amount, as explained above. Other choice could be to change the default value of the field, but, well ... Leave it as is.

Reply:
Well, I think it's illogical, since the default values are used for every shipping method except this one, then. When you tick the box, you are editing things, you see the amount field value as zero... of course people will edit that. Why 1000 then? Why not 99999? This has no sens to me... '^^

Reply:
Zero is not a good default value. The decision of 1000 was taken, I just stick to it. But I agree it shouldn't be changed on the demo data. I set this as the default value for the field `amount`.